docs/tsconfig.json

```diff
@@ -4,6 +4,10 @@
   "compilerOptions": {
     "baseUrl": "."
   },
-  "include": ["../sdk/src/*"], // Adjust the path as needed
-  "exclude": ["../sdk/src/index.ts"]
-}
+  "include": [
```

Comment:
@gregdhill what are these changes for? There's a build error relating to a missing module in the sdk directory—to fix it we we need to run `yarn` in the `sdk` directory. Not sure we should need to do that though. We can if we need to, but don't want to change the build settings without checking :)

Reply:
I'll also sort out access to the Vercel logs—not sure why you don't have that